.circleci/config.yml

```diff
@@ -85,6 +109,12 @@ commands:
       - run: echo 'nvm use 16.13.1' >> $BASH_ENV
       - run: node -v
 
+  set_up_git_user:
+    description: Set git config parameters
+    steps:
+      - run: git config --global user.email "dev+ci@revenuecat.com"
+      - run: git config --global user.name "CircleCI"
```

Comment:
👌 
We could even call it `RevenueCat CI`

Reply:
we can also set these as env variables, but it might be overkill for now

Reply:
I thought about it, but then we might have to validate that they're defined so we don't end up pushing with no user / email again?
I'll change it to `RevenueCat CI` 👍🏻